recipes/hyperion/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "0.9.7" %}
+
+package:
+  name: hyperion
+  version: {{version}}
+
+source:
+  fn: Hyperion-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/h/hyperion/Hyperion-{{version}}.tar.gz
+  md5: 9ab9f0341d83a2c7a1d996e7438ba8d0
+
+build:
+  number: 0
+  script:
+     - python setup.py install
+     - ./configure --prefix=$PREFIX
+     - make
+     - make install
```

Comment:
Usually when the build script is more than one line we prefer an actual script (`build.sh`) to keep things more organized.